src/types/appended.rs

```diff
@@ -0,0 +1,40 @@
+use imap_proto::UidSetMember;
+use std::fmt;
+
+/// Meta-information about a message, as returned by
+/// [`APPEND`](https://tools.ietf.org/html/rfc3501#section-6.3.11).
+/// Note that `APPEND` only returns any data if certain extensions are enabled,
+/// for example [`UIDPLUS`](https://tools.ietf.org/html/rfc4315).
+#[derive(Clone, Debug, Eq, PartialEq)]
+#[non_exhaustive]
+pub struct Appended {
+    /// The unique identifier validity value of the mailbox that the message was appended to.
+    /// See [`Uid`] for more details. Only present if server supports [`UIDPLUS`](https://tools.ietf.org/html/rfc4315).
+    pub uid_validity: Option<u32>,
+
+    /// The unique identifier value of the messages that were appended.
+    /// Only present if server supports [`UIDPLUS`](https://tools.ietf.org/html/rfc4315).
+    /// Contains only a single value unless the [`MULTIAPPEND`](https://tools.ietf.org/html/rfc3502) extension
+    /// was used to upload multiple messages.
+    pub uids: Option<Vec<UidSetMember>>,
+}
+
+#[allow(clippy::derivable_impls)]
+impl Default for Appended {
+    fn default() -> Appended {
+        Appended {
+            uid_validity: None,
+            uids: None,
+        }
+    }
+}
+
+impl fmt::Display for Appended {
```

Comment:
Why implement `Dispay` for this type? I don't think that's actually going to be all that useful compared to the `Debug` that's already derived.

Reply:
Right, I don't remember :innocent: . I think I used it for something at some point, but definitely no longer. Will remove. Do you prefer force pushed amendments or new commits?

Reply:
New commits are generally better for changes after the first review since GitHub is _terrible_ at rendering diffs across squashes :sweat_smile: